name.go

```diff
@@ -36,6 +36,7 @@ type Name struct {
 }
 
 const separator = ":"
+const LogicalClusterAnnotationKey = "tenancy.kcp.dev/cluster"
```

Comment:
```suggestion
const AnnotationKey = "tenancy.kcp.dev/cluster"
```

Also, you can create a `( )` group and put both of these constants in it.